src/Config/SessionConfig.php

```diff
@@ -87,10 +88,8 @@ class SessionConfig extends StandardConfig
      * Name of the save handler currently in use. This will either be a PHP
      * built-in save handler name, or the name of a SessionHandlerInterface
      * class being used as a save handler.
-     *
-     * @var null|string
      */
-    protected $saveHandler;
+    protected null|string|SaveHandlerInterface $saveHandler = null;
```

Comment:
this bc break, please only change `@var`

Reply:
reverted.